diff --git a/src/qt6ct-style/qt6ctproxystyle.cpp b/src/qt6ct-style/qt6ctproxystyle.cpp
--- a/src/qt6ct-style/qt6ctproxystyle.cpp
+++ b/src/qt6ct-style/qt6ctproxystyle.cpp
@@ -49,7 +49,7 @@ void Qt6CTProxyStyle::reloadSettings()

     if(style != m_style)
     {
-        setBaseStyle(QStyleFactory::create(style));
+        setBaseStyle(QStyleFactory::create(style) ?: new QCommonStyle);
         m_style = style;
     }
 }
